mne/defaults.py

```diff
@@ -7,9 +7,9 @@
 from copy import deepcopy
 
 DEFAULTS = dict(
-    color=dict(mag='darkblue', grad='b', eeg='k', eog='k', ecg='m', emg='k',
-               ref_meg='steelblue', misc='k', stim='k', resp='k', chpi='k',
-               exci='k', ias='k', syst='k', seeg='saddlebrown', dbs='seagreen',
+    color=dict(mag='#00008B', grad='b', eeg='k', eog='k', ecg='m', emg='k',
+               ref_meg='#4682B4', misc='k', stim='k', resp='k', chpi='k',
+               exci='k', ias='k', syst='k', seeg='#8B4513', dbs='#2E8B57',
```

Comment:
Let's revert these. You can make your code understand these by making use of 
```
mne/utils/check.py:def _to_rgb(*args, name='color', alpha=False):
```
which wraps matplotlib's color-name-to-rgb(a) functionality

Reply:
... we can add a `output='float' | 'hex'` to `_to_rgba` if that would help. I can push but it should be pretty simple like:
```
try:
    out =  func(*args)
except ...:
    ...
if output == 'hex':
    out = np.clip([255 * o for o in out], 0, 255).round().astype(int)
    out = '#' + ''.join(f'{o:02X}' for o in out)
return out
```

Reply:
Hm. pyqtgraph seems to accept the matplotlib-color-names out of the box in the newest release. Only (0.8, 0.8, 0.8) is a problem because it expects RGB-values as integers from 0-255 and not float from 0-1.

_to_rgb would have to be added in +40 places in `mne-qt-browser`. Do you think that might justify changing (0.8, 0.8, 0.8) to (204, 204, 204)?

Reply:
> EDIT: I see, for matplotlib only the floats from 0-1 are accepted. What about changing to lightgray?

I think in MNE we should not have to change how we deal with / think about colors -- `matplotlib color` mindset has been around for a long time and is in a lot of our code / docstrings etc.

> _to_rgb would have to be added in +40 places in mne-qt-browser

You've only changed 5-10 (?) instances here, so I think you only really need to convert the problematic ones that come from MNE.

Reply:
> I think in MNE we should not have to change how we deal with / think about colors
I understand that. But since pyqtgraph seems to support matplotlib color-names in general, the only change now in MNE-Python would have to be changing the default bad-color from (0.8, 0.8, 0.8) to `lightgray`. Or is there a reason why the default-bad-color has to be a RGB-tuple?
(we could of course just change the handling of the bad-color in mne-qt-browser to convert the float-RGB to int-RGB, but that seems a bit inconsistent to me, WDYT?)

>You've only changed 5-10 (?) instances here, so I think you only really need to convert the problematic ones that come from MNE.

The colors from MNE go directly into methods to create the traces, ch-axis-names, overview-bar-representation, annotation-colors. Thats where all the changes would come from in `mne-qt-browser`.

Reply:
> (we could of course just change the handling of the bad-color in mne-qt-browser to convert the float-RGB to int-RGB, but that seems a bit inconsistent to me, WDYT?)

To me the inconsistency comes from pyqtgraph not handling tuple-of-float automatically, as this is an accepted matpltolib color. So they support some matplotlib color types but not all.

Does `'0.8'` work? It should since it's an understood matplotlib color. If so let's just go with that. If it doesn't, I can live with the `'lightgray'` change

Reply:
Yes, `0.8` does work too